terraform/environments/core-network-services/monitoring.tf

```diff
@@ -24,7 +24,7 @@ module "pagerduty_route53" {
   depends_on = [
     aws_sns_topic.route53_monitoring
   ]
-  source                    = "github.com/ministryofjustice/modernisation-platform-terraform-pagerduty-integration?ref=v2.0.0"
+  source                    = "github.com/ministryofjustice/modernisation-platform-terraform-pagerduty-integration?ref=0179859e6fafc567843cd55c0b05d325d5012dc4"
```

Comment:
As above - can you add a comment with the tag version for readability?
PS - if you're still reading these, well done.